.eslintrc.js

```diff
@@ -1,5 +1,3 @@
-const reactPercy = require('./packages/eslint-plugin-react-percy');
-
 module.exports = {
```

Comment:
Very low priority, but you could consider converting this back to a plain JSON `.eslintrc` file now that you don't need this import.